### ISSUE TITLE
Berechtigungen Gremium/Mitglied

### ISSUE THREAD

**Issue by tschuepbach** [cevi/hitobito_cevi#190]
Wir haben immer wieder das "Problem", dass Mitglieder von Fachgruppen Anlässe (auf der Gruppe) erstellen möchten, aber nicht über die nötigen Berechtigungen dazu verfügen. Aktuell können beim Gruppentyp "Gremium" nur Personen mit der Rolle `Leitung` Anlässe erstellen.

Spricht etwas dagegen, der Rolle `Mitglied` folgende Berechtigungen zu geben?
<img width="269" alt="image" src="https://github.com/cevi/hitobito_cevi/assets/40494600/e7aa64d9-3d22-4381-b78f-974bd2f951ae">

Der aktuelle workaround ist, dass wir alle Personen als Leitung erfassen. Dies führt aber wieder zu Problemen bei Abos nach Rollen.

Ein weiterer, möglicher workaround wäre, dass immer die Leitung die Anlässe erstellt und die entsprechenden Personen bei Anlass als Hauptleitung hinzufügt. Aber dies ist aus meiner Sicht nicht gerade Ehrenamt-freundlich, resp. hängt so immer alles der Leitung der Fachgruppe.

**Comment by patrickuhlmann** [comment]
Die Fachgruppe International ist von diesem Problem aktuell betroffen. Einerseits laufen Bestrebungen alle Anlässe auf der cevi.db zu erfassen aber andererseits ist das ziemlich mühsam mit den aktuellen Berechtigungen. Ideal wäre wenn jedes Ressort (= jedes Mitglied) seine Anlässe erfassen und verwalten könnte.

Lassen sich die Berechtigungen eigentlich nicht einzeln vergeben (also das man der Rolle Mitglied nur die Schreibberechtigung für Anlässe gibt, nicht aber erstellen von Abos und Schreiben auf der Gruppe und darunterliegenden Gruppen)?

**Comment by nchiapol** [comment]
@patrickuhlmann 
Die Berechtigungen sind unter https://hitobito.readthedocs.io/de/latest/access_concept.html bzw. https://wiki.cevi.ch/index.php/Cevi.DB_Berechtigungen beschrieben.

Grundsätzlich gibt es bei Gruppen nur die Unterscheidung "lesen" oder "lesen und schreiben". Für alles Bearbeiten von Mitgliedern, Abos oder Anlässen sind Schreibrechte nötig.

Entsprechend: Nein, es lassen sich nur alle oder keine Schreibrechte vergeben.


Betreffend Rechten auf Anlässen sind dir Rechte mindestens aus meiner Erfahrung in der Region und als  Gremienleiter korrekt. Ich möchte nicht, dass jedes Mitglied meiner Gremien selbständig Anlässe oder Abos anlegt. Die Koordination von Anlässen und Kommunikationskanälen ist in meinen Augen eine Leitungsaufgabe - und falls das Erstellen delegiert werden soll, ist bei uns eigentlich in jedem Gremium eine Vertretung der GS dabei, die insbesondere als Unterstützung für administrative Aufgaben gedacht ist.

Wir können die Rechte auf Stufe Dachverband aber natürlich anders definieren als bei den MIOs - oder wir können in allen Gremien eine neue Rolle "Assistenz" einfügen die ebenfalls Schreibrechte hat. "Assistenz" wäre sicher die flexibelste Lösung.

**Comment by patrickuhlmann** [comment]
Ich würde die Lösung "Assistenz" begrüssen. Dann kann jede Fachgruppe das selber so regeln wie sie es möchten. Zudem ist es effektiv unschön allen die Gruppe Leitung zu vergeben da es dann für aussenstehende unklar ist wer effektiv die Leitung hat und auch Mailinglisten (z. B. an alle Leiter einer Fachgruppe) zu weit streuen.

**Comment by tschuepbach** [comment]
> Betreffend Rechten auf Anlässen sind dir Rechte mindestens aus meiner Erfahrung in der Region und als Gremienleiter korrekt. Ich möchte nicht, dass jedes Mitglied meiner Gremien selbständig Anlässe oder Abos anlegt. Die Koordination von Anlässen und Kommunikationskanälen ist in meinen Augen eine Leitungsaufgabe (...)

Da kenne ich einige Gremien, welche etwas anders funktionieren..
 
> (...) - oder wir können in allen Gremien eine neue Rolle "Assistenz" einfügen die ebenfalls Schreibrechte hat. "Assistenz" wäre sicher die flexibelste Lösung.

Das führt doch nur dazu, dass in Gremien, in welchen jetzt alle als Leitung erfasst sind, nebst ihren "normalen" Rollen, zusätzlich noch die Rolle "Assistenz" vergeben wird. Zwecks Nachvollziehbarkeit ist dies aus meiner Sicht zusätzlich verwirrend, z.B. im Profilverlauf oder wenn jemand kein Ahnung hat, wieso es die Rolle "Assistenz" gibt.

Ich persönlich begrüsse (Sicht GS Cevi Schweiz)
- Option 1: Die Rolle Mitglied erhält die gleichen Rechte, wie die "Leitung"
- Option 2: Option 1 nur auf Ebene Dachverband

Aus Sicht der Gremien in Bern muss ich kurz noch Rücksprache halten.

**Comment by patrickuhlmann** [comment]
Ich stimme zu das eine Rolle mit dem Namen "Assistenz" unklar sein könnte. Man könnte diese aber auch anders benennen. Beispielsweise "Eventmanager", "Eventadministration" oder "Anlassverwalter/in". Dann wäre es besser verständlich (auch im Profilverlauf). Etwas blöd daran ist natürlich das man durch die Rolle nicht nur Anlässe, sondern auch Mitglieder und Abos bearbeiten kann. Es wäre schon schöner wenn man dann auch nur die Schreibrechte für Anlässe erhält.

**Comment by patrickuhlmann** [comment]
Der referenzierte PR sollte allen Rollen auf den Gremien die Berechtigung geben Anlässe zu erstellen. Neue Rollen habe ich keine eingeführt.

**Comment by patrickuhlmann** [comment]
PR erstellt auf hitobito/hitobito_cevi https://github.com/hitobito/hitobito_cevi/pull/125
